.circleci/config.yml

```diff
@@ -1205,3 +1210,102 @@ workflows:
                 - accessibility-testing-repo
     jobs:
       - deploy-accessibility-testing
+
+  scheduled-daily-tests-content-repo:
```

Comment:
[pebble] This is effectively duplicating the `build-and-test` workflow. Is there a DRY way to approach this?

Reply:
Responded to this in Slack. The short answer to this is the code is currently DRY. We are only referencing the jobs already created.